vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -113,18 +113,23 @@ bool split_multi_example_and_stash_labels(const multi_ex& examples, ccb& data)
 // create empty/default cb labels
 void create_cb_labels(ccb& data)
 {
-  data.shared->l.cb.costs = data.cb_label_pool.get_object();
+  data.cb_label_pool.get_object(data.shared->l.cb.costs);
```

Comment:
I am struggling with a good name for `get_object` since one would expect it to return a value and not place the returned value on an out argument. Maybe `aquire_object`, `reclaim_object?`

Reply:
I like that naming